maisim/maisim.Game/Graphics/MaisimColour.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using maisim.Game.Beatmaps;
+using maisim.Game.Component;
+using osu.Framework.Extensions.Color4Extensions;
+using osuTK.Graphics;
+
+namespace maisim.Game.Graphics
+{
+    public class MaisimColour
+    {
+        /// <summary>
+        /// Get the colour for <see cref="TrackCard"/> and <see cref="TrackCardFocused"/> background colour.
+        /// </summary>
+        /// <param name="difficultyRating"><see cref="DifficultyRating"/> value</param>
+        /// <returns></returns>
+        /// <exception cref="ArgumentOutOfRangeException">Throw if the value is not right.</exception>
+
+        public Color4 GetDifficultyColor(DifficultyRating difficultyRating)
```

Comment:
Make this method static to not have to create a `MaisimColour` instance each time we want to calc the color associated to the difficulty rating.

Reply:
Yes, that's how the way I want to get the enum value from